### PR TITLE
Create git-ignore.gitignore

### PR DESCRIPTION
**Pull request by KitsuneAkuma**
ignore him

**he's bad:**

### CODE REVIEW

**Comment by KitsuneAkuma** [comment]
nvm don;t know what's happening

**Comment by Polda18** [comment]
I don't know what's the purpose of this commit. I'm afraid that's not going to work.

File: git-ignore.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+ignore me
```
Comment (Polda18) [inline git-ignore.gitignore:1]: What does that mean? Do you know what `.gitignore` file is used for?

**Comment by OsinniyApps** [comment]
spam

**Pull request closed without merging**